Fixes #3229 User settings in PKSim.CLI only enable a single core

rwmcintosh opened the pull request:
Fixes #3229

# Description
Enable more cores for snapshot evaluation

## Type of change

Please mark relevant options with an `x` in the brackets.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires documentation changes (link at least one [user](https://github.com/Open-Systems-Pharmacology/docs) or [developer](https://github.com/Open-Systems-Pharmacology/developer-docs) documentation issue):
- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
- [ ] Other (please describe):

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Integration tests
- [ ] Unit tests
- [x] Manual tests 
- [ ] No tests required

# Reviewer checklist

Mark everything that needs to be checked before merging the PR.

- [ ] Check if the code is well documented
- [ ] Check if the behavior is what is expected
- [ ] Check if the code is well tested
- [ ] Check if the code is readable and well formatted
- [ ] Additional checks (document below if any)
- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

# Screenshots (if appropriate):
<img width="666" height="886" alt="image" src="https://github.com/user-attachments/assets/f4d67bc3-856b-4718-9da3-4da3c3b668b8" />

# Questions (if appropriate):

rwmcintosh committed:
Fixes #3229 User settings in PKSim.CLI only enable a single core

rwmcintosh committed:
reset to default

rwmcintosh committed:
update core

msevestre commented on src/PKSim.R/Bootstrap/ApplicationStartup.cs:
```diff
@@ -73,7 +73,7 @@ private IContainer performInitialization()
       {
          container.Register<ICoreWorkspace, IWorkspace, CLIWorkspace>(LifeStyle.Singleton);
-         container.Register<ICoreUserSettings, CLIUserSettings>();
+         container.Register<ICoreUserSettings, CLIUserSettings>(LifeStyle.Singleton);
```
are we exposing this somehow to the outside ?

msevestre reviewed (approved):
UserSettings instance registered as singleton but no instance is ever exposed for modification,. Probably makes sense anyways to have it as a singleton but the net effect is 0 right now

msevestre commented:
@rwmcintosh Build fails

rwmcintosh committed:
add some chart exports to png

rwmcintosh commented:
> @rwmcintosh Build fails

Yeah, I updated core and there was some updates needed. I had implemented export charts to png in all places where it was possible to copy to clipboard. There were two instances that needed the PNG export implemented.
